# HG changeset patch
# Parent 188b070e1206131b37241ddd787df8df90c1033c
# User Frédéric Wang <fred.wang@free.fr>
MathML Operator Dictionary: remove obsolete infix entries for coproduct, wedge and vee (bug 534970). r=karlt

diff --git a/layout/mathml/mathfont.properties b/layout/mathml/mathfont.properties
--- a/layout/mathml/mathfont.properties
+++ b/layout/mathml/mathfont.properties
@@ -1213,30 +1213,27 @@ operator.\u20D0.postfix = lspace:0 rspac
 operator.\u20D1.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D1;
 operator.\u20D6.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D6;
 operator.\u20D7.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20D7;
 operator.\u20E1.postfix = lspace:0 rspace:0 stretchy accent direction:horizontal # &#x20E1;
 operator.\u2190\u200B.infix = lspace:5 rspace:5 # &ShortLeftArrow;
 operator.\u2191\u200B.infix = lspace:2 rspace:2 # &ShortUpArrow;
 operator.\u2192\u200B.infix = lspace:5 rspace:5 # &ShortRightArrow;
 operator.\u2193\u200B.infix = lspace:2 rspace:2 # &ShortDownArrow;
-operator.\u2210.infix = lspace:3 rspace:3 direction:vertical # &Coproduct;
 operator.\u2223.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &VerticalBar;
 operator.\u2223.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &VerticalBar;
 operator.\u2225.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &DoubleVerticalBar;
 operator.\u2225.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &DoubleVerticalBar;
 operator.\u2282\u020D2.infix = lspace:5 rspace:5 # &NotSubset;
 operator.\u2283\u020D2.infix = lspace:5 rspace:5 # &NotSuperset;
 operator.\u228E.prefix = lspace:1 rspace:2 largeop movablelimits symmetric direction:vertical # &UnionPlus;
 operator.\u2295.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CirclePlus;
 operator.\u2296.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CircleMinus;
 operator.\u2297.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CircleTimes;
 operator.\u2299.prefix = lspace:0 rspace:3 largeop movablelimits symmetric # &CircleDot;
-operator.\u22C0.infix = lspace:3 rspace:3 direction:vertical # &Wedge;
-operator.\u22C1.infix = lspace:3 rspace:3 direction:vertical # &Vee;
 operator.\u23B0.prefix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &lmoustache; &lmoust;
 operator.\u23B1.postfix = lspace:0 rspace:0 stretchy fence symmetric direction:vertical # &rmoustache; &rmoust;
 operator.\u2500.infix = lspace:0 rspace:0 stretchy direction:horizontal # &HorizontalLine;
 operator.\u25A1.prefix = lspace:0 rspace:2 # &Square;
 operator.\u2606.infix = lspace:3 rspace:3 # &star;
 operator.\u2AC5\u0338.infix = lspace:5 rspace:5 # &nsubseteqq;
 operator.\u2AC6\u0338.infix = lspace:5 rspace:5 # &nsubseteqq;
 operator.\u2AEC.prefix = lspace:0 rspace:5 # &Not;
